### PR TITLE
Status filter for the RSS feed

### LINKED ISSUE #101: Create a subdashboard specifically for deprecated and removed features
Goal is to have an easy, focused place for developers to get information about what features are going to be removed from Chrome.

Context: https://groups.google.com/a/chromium.org/d/msg/blink-dev/nNsZDn_utUc/clT3aVUGZaAJ

### PR DESCRIPTION
**Pull request by jeffposnick**
@ebidel @beaufortfrancois

This exposes a new RSS feed, https://www.chromestatus.com/features?status=Deprecated,Removed (or any other comma-separated list of `impl_status_chrome ` values). It's a little hacky since I don't believe App Engine's datastore allows for `OR`s in its queries. I'm also only doing this for the `impl_status_chrome` field—it could be made more generic, but I'm not sure if there's value in that.

This together with the existing https://www.chromestatus.com/features#deprecated|removed may be sufficient to close #101

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -110,29 +110,34 @@ def get(self, path, feature_id=None):
...
-              break
-
-        feature_list = models.Feature.get_all( # Memcached
-            limit=max_items,
-            filterby=filterby,
-            order='-updated')
+        status = self.request.get('status', None)
+        if status:
+          statuses = status.split(',')
+          feature_list = models.Feature.get_all_with_statuses(statuses)
```
Comment (ebidel) [inline server.py]: feature_list = models.Feature.get_all_with_statuses(status.split(','))

File: server.py [hunk 2]
```diff
@@ -110,29 +110,34 @@ def get(self, path, feature_id=None):
...
-            normalized = normalized_name(v)
-            if category == normalized:
-              filterby = ('category =', k)
-              break
-
-        feature_list = models.Feature.get_all( # Memcached
-            limit=max_items,
-            filterby=filterby,
-            order='-updated')
+        status = self.request.get('status', None)
```
Comment (ebidel) [inline server.py]: remove

**Comment by ebidel** [comment]
Nice! I was going to take a look at this today too. Glad I didnt :+1: 

LGTM. just the nit.

File: server.py (hunk 2 above)
Comment (jeffposnick) [inline server.py]: Remove what? `status = self.request.get('status', None)` still seems necessary.

Comment (ebidel) [inline server.py]: Hmm, not sure. I think I was on a slow connection and this wasn't intended for here. Ignore.

Comment (ebidel) [inline server.py]: I think it was for `status = status.split(',')` line which you've inlined now.

**Comment by ebidel** [comment]
LGTM

**Pull request merged**